app/guide/template.njk

```diff
@@ -18,7 +18,7 @@
       </div>
     </div>
     {% for section in sections %}
-    <div class="nhsuk-grid-row" data-test-id="guide-section-{{section.id}}">
+    <div class="nhsuk-grid-row" data-test-id="guide-section-{{section.id}}" id="{{ section.id }}">
```

Comment:
Sorry... really picky but could we put the id right at the start please rather than at the end